cobalt/h5vcc/h5vcc_settings.cc

```diff
@@ -67,35 +65,69 @@ bool H5vccSettings::Set(const std::string& name, int32 value) const {
   const char kUpdaterMinFreeSpaceBytes[] = "Updater.MinFreeSpaceBytes";
 #endif
 
-  if (name == kDisableMediaCodec &&
-      value < static_cast<int32>(kDisableCodecCombinations.size())) {
-    can_play_type_handler_->SetDisabledMediaCodecs(
-        kDisableCodecCombinations[value]);
+  if (name == kMediaCodecAllowList && value.IsType<std::string>() &&
+      value.AsType<std::string>().size() < 256) {
+    std::vector<std::string> allowed_media_codecs =
+        base::SplitString(value.AsType<std::string>(), ";",
+                          base::TRIM_WHITESPACE, base::SPLIT_WANT_NONEMPTY);
+    // Allow only the codecs in the list.
```

Comment:
We should explicit mention that the allow list is based on string matching, and also add an example for avc (since "h264" won't work).

Reply:
Fixed.